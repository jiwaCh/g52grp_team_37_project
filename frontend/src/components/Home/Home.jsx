import React from 'react';
import { Typography } from '@material-ui/core';
import Container from '../Container';
// import styles from './Home.module.scss';

const Home = () => {
  return (
    <>
      <Container>
        <Typography>Hello World from Home</Typography>
      </Container>
    </>
  );
};

export default Home;
