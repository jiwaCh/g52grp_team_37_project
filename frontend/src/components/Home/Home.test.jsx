import React from 'react';
import Home from './Home';
import toJson from 'enzyme-to-json';
import { shallow } from 'enzyme';

it('renders correctly', () => {
  const component = shallow(<Home />);
  expect(toJson(component)).toMatchSnapshot();
});
