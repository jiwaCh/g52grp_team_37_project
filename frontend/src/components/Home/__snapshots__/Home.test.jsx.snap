// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<Fragment>
  <Container>
    <WithStyles(ForwardRef(Typography))>
      Hello World from Home
    </WithStyles(ForwardRef(Typography))>
  </Container>
</Fragment>
`;
